WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1229,11 +1243,35 @@ AttachBriefcaseItemIcon     = height:28px; width:28px;
 ###################
 #   CONTACTS-APP SPECIFIC STUFF
 ###################
+ContactViewOutsidePadding   = @ComposeViewOutsidePadding@
+ContactEditInputSpacing     = 26px;
+ContactEditButtonHeight     = @FormButtonHeight@
```

Comment:
Have kept it here to have it customised. Have added a separate section for Form and Contact form is inheriting those properties. So trying to keep a structure and having placeholders.